### PR TITLE
feat: expense list page

### LINKED ISSUE #5: Frontend: Expense list page
A page where we can display list of expenses (or income) made by Teknologi Umum. Refer to https://opencollective.com/vuejs/expenses?direction=RECEIVED&collectiveSlug=vuejs

Be very careful when displaying money. For data, see #4

### PR DESCRIPTION
**Pull request by qepo17**
image

### CODE REVIEW

File: app/components/expense-filter.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const ExpenseFilter = () => {
+  return (
+    <div className="py-4 flex justify-between items-center">
+      <div>
+        <ButtonPrimary name="Received" />
+        <ButtonPrimary name="Submmitted" />
+      </div>
+      <div className="flex space-x-4">
+        <input type="text" placeholder="Search" className="input" />
```
Comment (yogip07) [inline app/components/expense-filter.tsx]:
```suggestion
        <input type="text" placeholder="Search" className="input border rounded-lg p-2" />
```
Enhance search input a bit by adding a border and padding

**Review by yogip07 — APPROVED** [review]
gokil

File: app/components/expense-filter.tsx (hunk 1 above)
Comment (qepo17) [reply app/components/expense-filter.tsx]: updated

File: app/root.tsx [hunk 2]
```diff
@@ -6,6 +6,11 @@ import {
   ScrollRestoration,
 } from "@remix-run/react";
 
+import type { LinksFunction } from "@remix-run/node";
+import styles from "./tailwind.css?url";
```
Comment (ridhof) [inline app/root.tsx:10]: ini memang ada `?url` begitu kah pak?

Comment (qepo17) [reply app/root.tsx:10]: @ridhof yap, di step 5 https://tailwindcss.com/docs/guides/remix

**Comment by aldy505** [comment]
@qepo17 ada conflict

File: app/components/navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { ButtonLink } from "./button-link";
+
+const Navbar = () => {
+  return (
+    <nav className="flex justify-between items-center py-4 px-4 bg-white shadow-md">
+      <div>
+        {/* <img src="/path/to/logo.png" alt="Logo" className="h-8" /> */}
```
Comment (aldy505) [inline app/components/navbar.tsx:7]: Should be removed rather than just commented out

File: app/components/expense-sidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const ExpenseSidebar = () => {
+  return (
+    <aside className="p-4 bg-white rounded-md">
+      <div className="mb-4">
+        <h2 className="text-lg font-bold">Collective Balance</h2>
+        <p className="text-xl font-thin">
+          {new Intl.NumberFormat("id-ID", {
+            style: "currency",
+            currency: "IDR",
+          }).format(10000)}
```
Comment (aldy505) [inline app/components/expense-sidebar.tsx:10]: Should be a prop later on

File: app/components/expense-list.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        method: "paypal",
+      },
+    },
+  ];
+
+  return (
+    <CardExpense
+      title="Recent Expenses"
+      description="Here are the recent expenses made by the collective."
+      expenses={expenses}
```
Comment (aldy505) [inline app/components/expense-list.tsx:40]: Should be a props later on

**Review by aldy505 — APPROVED** [review]
Speaking of, should we do i18n with all these?

**Pull request merged**